src/main/kotlin/com/launchdarkly/intellij/LaunchDarklyApiClient.kt

```diff
@@ -33,7 +37,7 @@ class LaunchDarklyApiClient() {
         @JvmStatic
         fun projectInstance(apiKey: String? = null, baseUri: String? = null): ProjectsApi {
```

Comment:
I realized that `projectInstance` and `flagInstance` functions are both virtually the same code with just loc difference which is the last return line. Can we please DRY this so we can maintain the code easier? Thank you.